snippets/component.js.json

```diff
@@ -2,9 +2,9 @@
   "Solid Functional Component": {
     "prefix": "scomp",
     "body": [
-      "export default function ${1:${TM_FILENAME_BASE}}() {\n\n\treturn (\n\t\t<div>${1:${TM_FILENAME_BASE}}</div>\n\t);\n}"
+      "\n function ${1:${TM_FILENAME_BASE}}() {\n\n\treturn (\n\t\t<div>${1:${TM_FILENAME_BASE}}</div>\n\t);\n}\n\texport default ${1:${TM_FILENAME_BASE}};"
     ],
-    "description": "Base for an empty solid.js component"
+    "description": "Base for an empty solid.jsx component"
```

Comment:
What's the point of this one? Would rather change to to "Solid" or "SolidJS"

Reply:
I thought it was referring to the name and the extension of the component. solid(SolidJS).jsx since we agreed to not use  .js  but .jsx , my bad, let's use  use SolidJS.